src/Contrib/Authentication/OpenIdConnect/Masa.Contrib.Authentication.OpenIdConnect.EFCore/Repositories/ClientRepository.cs

```diff
@@ -46,6 +46,7 @@ public async Task<PaginatedList<Client>> GetPaginatedListAsync(int page, int pag
                     .Include(c => c.AllowedCorsOrigins)
                     .Include(c => c.ClientSecrets)
                     .Include(c => c.AllowedScopes)
+                    .AsSplitQuery()
```

Comment:
I don't think it needs split query, you just need control left join and inner join.